Login.spec.js

```diff
@@ -0,0 +1,105 @@
+describe('Login Page Tests', () => {
+
+  before(() => {
+    // Visit the Login page before running the tests
+    cy.visit('/login');
+  });
+
+  beforeEach(() => {
+    // Clear local storage before each test
+    window.localStorage.clear();
+  });
+
+  it('Should render the login page correctly', () => {
+    // Verify if the logo is displayed
+    cy.get('img[alt="ATS Logo"]').should('be.visible');
+
+    // Check the main headings
+    cy.contains('ATS').should('be.visible');
+    cy.contains('Login to Attendance Tracker System (ATS)').should('be.visible');
+  });
+
+  context('Form Submission Tests', () => {
+
+    it('Should show validation errors for empty fields', () => {
+      // Click the login button without filling the form
+      cy.contains('button', 'Login').click();
+
+      // Check for validation messages
+      cy.contains('Email is required').should('be.visible');
+      cy.contains('Password is required').should('be.visible');
+    });
+
+    it('Should show validation error for invalid email', () => {
+      // Enter invalid email
+      cy.get('input[type="email"]').type('invalid-email');
+      cy.get('input[type="password"]').type('password123');
+      
+      // Click login
+      cy.contains('button', 'Login').click();
+
+      // Check for validation message
```

Comment:
Consider creating a helper function to validate error messages. This function can take the expected error message as a parameter and check if it is visible. This will make the tests more modular and maintainable.